homeassistant/components/roborock/image.py

```diff
@@ -55,13 +73,15 @@ def __init__(
         map_flag: int,
         starting_map: bytes,
         map_name: str,
+        sizes: dict[Size, float],
+        drawables: list[Drawable],
```

Comment:
I plan to add 2 more parameters here.

WDYT about either passing the entire `config_entry`, or passing a `Sizes` instance directly?

Reply:
I think we should extract everything from the config entry and then pass it to the ImageEntity. I do agree on your sizes comment though